runner.js

```diff
@@ -1,42 +1,59 @@
-const fs = require('fs')
+const fs = require('fs').promises
 const cp = require('child_process')
-const util = require('util')
 const ulid = require('ulid')
 
-const writeFile = util.promisify(fs.writeFile)
-const deleteFile = util.promisify(fs.unlink)
+const buildArgs = (config, name) => {
+  const { image, cmd, extension } = config
+  return [
+    'run',
+    '--rm',
+    '--name',
+    name,
+    '-v',
+    `${process.cwd()}:/usr/src/app`,
+    '-w',
+    '/usr/src/app',
+    image,
+    cmd,
+    `${name}${extension}`
+  ]
+}
 
-const buildArgs = (name) => [
-  'run',
-  '--rm',
-  '--name',
-  name,
-  '-v',
-  `${process.cwd()}:/usr/src/app`,
-  '-w',
-  '/usr/src/app',
-  'node:12-alpine',
-  'node',
-  `${name}.js`
-]
+const parseLanguage = (language) => {
+  return new Promise((resolve, reject) => {
+    fs.readFile('./languages.json').then(contents => {
```

Comment:
can we just import the json file at the top ? require('./languages.json')

Reply:
fixed